test/solvers/advanced/proximal_operator_test.py

```diff
@@ -230,7 +230,7 @@ def test_proximal_convconj_l2_sq_wo_data():
     x = space.element(x_arr)
 
     # Create data
-    g = space.element(-2 * x_arr)
+    g = space.zero()
```

Comment:
Because the naming of the test is ```l2_sq_wo_data```, and I assume that wo is suppose to mean "with out". And yes, it is still doing something. It is testing the ```l2_sq``` when there is "no off-set".

The test just below it in the code is identical if you don't do this change.